packages/cockpit/src/transactions/conversion/boleto/result.js

```diff
@@ -0,0 +1,43 @@
+import {
+  always,
+  apply,
+  applySpec,
+  divide,
+  equals,
+  ifElse,
+  juxt,
+  last,
+  multiply,
+  path,
+  pipe,
+} from 'ramda'
+
+const total = path(['aggregations', 'boleto', 'doc_count'])
+
+const paid = path(['aggregations', 'boleto', 'paid', 'doc_count'])
+
+const conversion = pipe(
+  juxt([
+    paid,
+    total,
+  ]),
+  ifElse(
+    pipe(
+      last,
+      equals(0)
+    ),
+    always(100),
+    pipe(
```

Comment:
Essa função também pode ser separada em outra, de nome `getPercentage`:

```js
const getPercentage = pipe(
  apply(divide),
  multiply(100)
)
```

Ficando a chamada:

```suggestion
    getPercentage
```